src/components/NewLocationPage/ChartFooter/ChartFooter.style.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+import { TooltipAnchorText } from 'components/InfoTooltip/Tooltip.style';
+
+export const Footer = styled.div`
+  display: flex;
+  flex-direction: column;
+  color: ${COLOR_MAP.GRAY_BODY_COPY};
+`;
+
+export const FooterSection = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: row;
+`;
+
+export const FooterText = styled.p`
+  margin: 0 0 0.75rem 0;
+`;
+
+export const SingleButtonWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+`;
+
+export const ButtonContainer = styled.div`
+  flex-grow: 0;
+  margin-left: 0.5rem;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    margin-left: 7.5rem;
+  }
+`;
+
+export const LegendContainer = styled.div`
```

Comment:
i think we can use [the current LegendContainer](https://github.com/covid-projections/covid-projections/pull/3670/files#diff-d2b8fc3b1fc0390125ed3994eec3c2aff7151f7e3a572c0bc60dc06296372665R73)